screens/Contacts/ContactGraph/ContactGraph.tsx

```diff
@@ -1,12 +1,95 @@
-import { View, Text } from "react-native"
+import { View, TextInput, TouchableWithoutFeedback, Keyboard } from "react-native"
 import { styles } from "./styles"
+import Graph, { Node } from "../../../components/Graph/Graph"
 
-const ContactGraph = () => {
+import ForceDirectedGraph from "../../../components/Graph/ForceDirectedGraph/ForceDirectedGraph"
+import { useState } from "react"
+
+import NodeModal from "../../../components/Graph/NodeModal/NodeModal"
+
+const graph = new Graph(
+  ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10"],
+  ["0", "2", "3", "4", "5", "6", "7", "8", "9", "10"],
+  ["1", "1", "1", "1", "1", "1", "2", "2", "3", "3"],
+  [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
+)
```

Comment:
Please don't use example code in the final codebase. You can maybe create a separate file called `mocks.ts` for the component so that testing parts remain isolated and can be easily deleted when no longer necessary.